pkg/controller/plan/validation.go

```diff
@@ -432,6 +434,14 @@ func (r *Reconciler) validateVM(plan *api.Plan) error {
 		Message:  "VM has more than one interface mapped to the pod network.",
 		Items:    []string{},
 	}
+	missingStaticIPs := libcnd.Condition{
+		Type:     VMMissingGuestIPs,
+		Status:   True,
+		Reason:   MissingGuestInfo,
+		Category: Critical,
+		Message:  "VM is missing Static IPs, please try to boot the VM first.",
```

Comment:
"VM is missing static IPs" doesn't reflect the problem good enough (it sounds like the VM is not configured with static IPs which is ok), how about
```suggestion
		Message:  "Guest information on vNICs is missing, cannot preserve static IPs. Make sure VMware tools are installed and the VM is running.",
```